front-end/src/components/Testimonials.js

```diff
@@ -21,21 +21,23 @@ const Testimonials = () => {
 					return (
 						<div key={testimonial.id} style={{ flex: 1 }}>
 							<span>
```

Comment:
No need for the additional span tag here. Also do you have the prettier extension installed because there seems to be a lot of extra spacing in your lines